.github/workflows/tests.yml

```diff
@@ -82,6 +82,12 @@ jobs:
           pip uninstall pennylane -y
           pip install git+https://github.com/PennyLaneAI/pennylane.git
 
+      - name: Install ML libraries for interfaces
```

Comment:
The only concern here is the fact that we are using the latest versions of `torch`, `jax`, and `tf`. I'd suggest to use some env variables such as `TORCH_VERSION`, `JAX_VERSION`, and `TF_VERSION` to have more control over the ML libs. It would be helpful to catch bugs related to some specific versions in the future too.

Reply:
Maybe we factor tests out into an LTS (ie oldest supported version) and the most recent?

Reply:
Even better!

Reply:
I just found only Pytorch explicitly supports the LTS version. I can use a solution from Ali that is currently used in PL.